src/main/assembly/dist/cfg/application.properties

```diff
@@ -11,5 +11,13 @@ ldap.provider.url=ldap://localhost
 # TODO: However that will require some extra work on the local test-vm
 ldap.users-entry=ou=users,ou=easy,dc=dans,dc=knaw,dc=nl
 
-# 64*1024*1024
-max-fileSize-toExtract-content-from=67108864
\ No newline at end of file
+#
+# The service will try to have Solr index extracted content from the file. This is limited to the
+# media types and file size configured below.
+#
+file-content-extraction.max-filesize=67108864
+file-content-extraction.mime-types=\
+  text/plain, \
```

Comment:
@PaulBoon , @jo-pol : do we need more default media types here?

Reply:
Confusing comment. This file *is* `application.properties`. You are probably referring to `debug-config/application.properties` specifically. I'll add pdf and make this properties the same in both files.